### PR TITLE
[PR-14653] Add docs with examples of PCP encryption

### PR DESCRIPTION
**Pull request by FidelioGorau**
Docs update for https://github.com/talkable/talkable/pull/6883

## Demo

http://void-docs.talkable.com/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-14653)

### CODE REVIEW

File: source/advanced_features/params_encryption.rst [hunk 1]
```diff
@@ -131,7 +131,10 @@ Please modify the front-end using this pseudo code example:
      _talkableq.push(['authenticate_customer', {
        email: '<%= to_json(TalkableEmail.encrypt(current_user.email)) %>',
        first_name: '<%= to_json(current_user.first_name) %>',
-       last_name: '<%= to_json(current_user.last_name) %>'
+       last_name: '<%= to_json(current_user.last_name) %>',
+       person_custom_properties: {
+         secret: '<%= to_json(current_user.encrypted_secret) %>'
```
Comment (zhuravel) [inline source/advanced_features/params_encryption.rst:136]:
```suggestion
         secret: '<%= to_json(TalkableEmail.encrypt(current_user.secret)) %>'
```

Comment (zhuravel) [reply source/advanced_features/params_encryption.rst:136]: `TalkableEmail` should be updated to a modern name as well

**Review by dgtv — APPROVED** [review]
LG
But do you know the reason of this blank space? Looks like this page is not only one case cc @zhuravel 
image
image
image

File: source/advanced_features/params_encryption.rst [hunk 2]
```diff
@@ -129,9 +129,12 @@ Please modify the front-end using this pseudo code example:
 
    <script>
      _talkableq.push(['authenticate_customer', {
-       email: '<%= to_json(TalkableEmail.encrypt(current_user.email)) %>',
+       email: '<%= to_json(TalkableEncodeService.encrypt(current_user.email)) %>',
```
Comment (zhuravel) [inline source/advanced_features/params_encryption.rst:132]: Encoding ≠ encrypting.
```suggestion
       email: '<%= to_json(TalkableEncryptionService.encrypt(current_user.email)) %>',
```

File: source/advanced_features/params_encryption.rst [hunk 3]
```diff
@@ -129,9 +129,12 @@ Please modify the front-end using this pseudo code example:
...
        first_name: '<%= to_json(current_user.first_name) %>',
-       last_name: '<%= to_json(current_user.last_name) %>'
+       last_name: '<%= to_json(current_user.last_name) %>',
+       person_custom_properties: {
+         secret: '<%= to_json(TalkableEncodeService.encrypt(current_user.secret)) %>'
```
Comment (zhuravel) [inline source/advanced_features/params_encryption.rst:136]:
```suggestion
         secret: '<%= to_json(TalkableEncryptionService.encrypt(current_user.secret)) %>'
```

**Review by zhuravel — APPROVED** [review]

**Comment by zhuravel** [comment]
@dgtv Good catch. The gaps are caused by the upgrade to Sphinx 3.
This PR didn’t cause the issue and can be merged, gaps will be fixed in a separate PR.

@pustovalov Looks like #223 didn’t fix all the issues, there are still gaps on other pages:
https://docs.talkable.com/advanced_features/email_encryption.html

File: source/advanced_features.rst [hunk 4]
```diff
@@ -31,7 +31,7 @@ each with links to integration details.
      - Explore full details for each individual customer and their referral journey,
        manage their rewards and deliver excellent customer service.
 
-   * - :ref:`Email encryption <advanced_features/email_encryption>`
+   * - :ref:`Params encryption <advanced_features/params_encryption>`
```
Comment (zhuravel) [inline source/advanced_features.rst:34]: Don’t forget to update the link to docs in the main repo:
https://github.com/talkable/talkable/blob/75993b84773fcb31ceb7c14d0819d30351aa74af/app/javascript/components/sites/site_edit/site_edit_general/integration_fields.jsx#L81

**Pull request merged**